sce_presubmit_handler.py

```diff
@@ -0,0 +1,43 @@
+from subprocess import check_output, PIPE
```

Comment:
instad of `PIPE` just use `subprocess.PIPE` instead
```suggestion
from subprocess import check_output
```